core/src/main/java/software/amazon/awssdk/util/json/Jackson.java

```diff
@@ -22,7 +22,6 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.ObjectWriter;
-import java.io.File;
 import java.io.IOException;
 import java.io.Writer;
 import software.amazon.awssdk.SdkClientException;
```

Comment:
In favor of renaming this JacksonUtils and making a final class with private constructor.

Reply:
👍